localstack/services/ses/ses_listener.py

```diff
@@ -0,0 +1,29 @@
+import re
+import xmltodict
+from localstack.utils.common import to_str
+from localstack.utils.aws.aws_responses import make_error
+from localstack.services.generic_proxy import ProxyListener
+
+
+class ProxyListenerSES(ProxyListener):
+    def forward_request(self, method, path, data, headers):
```

Comment:
nitpick: This is the default behavior in the `ProxyListener` base class - the `forward_request` method could be removed from here if we don't need any custom logic in it.